### PR TITLE
Choices and Tabs Updates

### PR DESCRIPTION
**Pull request by philschanely**
## Description

- Adds some variations to Choices: arrow, icon, and radio.
- Further improves tab and tab pane spacing settings.
- Addresses a bug with arrow navigation between tabs and choices.

Changes drafted in `products` https://github.com/Kajabi/kajabi-products/pull/15888

### Screenshots

Screen Shot 2020-10-02 at 11 17 58 AM

Screen Shot 2020-10-02 at 11 17 38 AM

### CODE REVIEW

**Review by joe-dakroub — APPROVED** [review]
These changes look great! Thank you for fixing the keyboard issues as well!! Nice new style additions. No concerns. 👍

File: lib/sage-frontend/stylesheets/system/patterns/elements/_choice.scss [hunk 1]
```diff
@@ -20,40 +21,84 @@ $item-radio-indicator: "::after";
   color: $sage-tabs-default-color;
   border: sage-border();
   border-radius: sage-border(radius);
-  transition: color $sage-transition, background-color $sage-transition, box-shadow $sage-transition;
+  transition: color $sage-transition, background-color $sage-transition, outline-color $sage-transition, box-shadow $sage-transition;
```
Comment (teenwolfblitzer) [inline lib/sage-frontend/stylesheets/system/patterns/elements/_choice.scss]: Minor detail; with multiple properties listed, would this be more readable as:

```
transition: $sage-transition;
transition-property: color…
```

**Review by teenwolfblitzer — APPROVED** [review]
👍 LGTM, great work! One non-blocking item to consider.

File: lib/sage-frontend/stylesheets/system/patterns/elements/_choice.scss (hunk 1 above)
Comment (philschanely) [reply lib/sage-frontend/stylesheets/system/patterns/elements/_choice.scss]: Yes. Yes it would.

**Pull request merged**